docs/insights/scope.md

```diff
@@ -21,6 +21,7 @@ Access the **Insights** menu from the left-side navigation menu of our web app.
 |Test Overview|Shows a variety of views of the data related to all the tests executed that match the specified filter criteria broken into three focus tabs. <br/>The **Overview** tab shows concatenated pass/fail statistics and platform breakdowns. <br/>The **Tests** tab shows a table of all tests run on either **Virtual Cloud** or **Real Devices**, which you can sort by any of the data columns. You can click any test to see a historical breakdown of that test, and you can also export the data to a local CSV file. <br/>The **Errors** tab shows the total number of errors across the execution of all tests in the filter for either **Virtual Cloud** or **Real Devices** along with a graph depicting the error rate over time. Below, there is a breakdown of each error with the number of times it occurred in the period, which you can click any error to see the list of tests in which it occurred.|
 |Test History| Shows a visual snapshot of the results for a specific test over time. See the [Test History](/insights/history) page for specific views and capabilities descriptions.|
 |Trends| Shows graphical visualizations of all tests. Applying filters to this view makes it easy to compare test outcomes for different variables, such as device browser version. See the [Trends](/insights/trends) page for detailed documentation.|
+|Coverage| <small><span className="sauceDBlue">Org Admin Only</span></small> Shows the breakdown of browsers and devices against which you've run tests, giving you an idea of how comprehensively you are testing across different platforms.|
```

Comment:
we could add in addition to this to show the benefit something along the lines of:
giving you an idea... and informing improvents to your testing strategy.

We've seen customers test on old devices, or devices not used in the market they are operating in. So this dataset can help them improve their testing strategy.